test/index-core.test.js

```diff
@@ -47,4 +47,14 @@ test('diverging indicator index-core', ()=>{
   const simpleIndex = indexCore(simpleIndicators, simpleEntities);
   expect(simpleIndex.indexedData['Tigris and Eurphrates']['2.3']).toBe(70);
   expect(simpleIndex.indexedData['Twilight Imperium']['2.3']).toBe(70);
+})
+
+test('indicator overide index-core', ()=>{
```

Comment:
does this test do what it's suppose to ? it's not forcing any specific value to be passed manually

Reply:
the value which is forced is in the data so the line 
`expect(simpleIndex.indexedData['Catan']['1']).toBe(10);` 
is the one which tests whether the forcing is happening the previous lines test that the default behaviour is unaffected

We expect Catan 1 should be 10 because it's the value set explicitly in the spreadsheet ...

https://github.com/signal-noise/index-core/blob/857bcaf41dad102e653ecc06cf0a365e4487dc92/data/simple-index-set/entities.csv#L2